SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+The Substra team and community take security issues very seriously.
+
+We appreciate your efforts to responsibly disclose your findings and we will make our best to acknowledge your contributions.
+
+## Reporting a Vulnerability
+
+> Note: Please do not report security vulnerabilities through public Github issues.
+
+To report a securty issue, please send an email at [security@substra.org](mailto:security@substra.org?subject=SECURITY) including the word "SECURITY" in the subject line.
```

Comment:
This email is @substra.org, but we also use substrafoundation.github.io and substra.ai. 

It be better for our communication to use always the same domain name, but I'm not sure it is possible.

Reply:
Good point!

Reply:
Done, with the precious help from @bowni !